### PR TITLE
Added snowflake, Roche and SUBCOM

### PR DESCRIPTION
**Pull request by tchan102**

### CODE REVIEW

File: README.md
Comment (ewang2002) [inline README.md]: SUBCOM concerns me. Specifically, they are 
> Only considering candidates within 60 miles of Newington, NH

There is no mention of relocation anywhere, so I'm tempted to just remove SUBCOM altogether.

I'll let the others decide what to do with SUBCOM.

**Review by ewang2002 — CHANGES_REQUESTED** [review]
See my one comment.

I'll let the others decide what should be done about SUBCOM.

File: README.md
Comment (ewang2002) [reply README.md]: Oops, I meant to select "Comment" not "Request Changes." Don't worry about making any changes for now.

**Comment by tchan102** [comment]
deleted the SUBCOM need another review to pass the merge

**Review by ewang2002 — APPROVED** [review]
Looks good, thank you!

**Comment by ewang2002** [comment]
> deleted the SUBCOM need another review to pass the merge

Feel free to request another review from me if I do review your PR and request changes.

**Pull request merged**
